selfspy/__init__.py

```diff
@@ -97,10 +97,14 @@ def check_with_encrypter(password):
     lockname = os.path.join(args['data_dir'], cfg.LOCK_FILE)
     cfg.LOCK = LockFile(lockname)
     if cfg.LOCK.is_locked():
-        print '%s is locked! I am probably already running.' % lockname
-        print 'If you can find no selfspy process running, it is a stale lock and you can safely remove it.'
-        print 'Shutting down.'
-        sys.exit(1)
+        if os.path.exists(lockname):
+            print '%s is locked! I am probably already running.' % lockname
+            print 'If you can find no selfspy process running, it is a stale lock and you can safely remove it.'
+            print 'Shutting down.'
+            sys.exit(1)
+        else:
+            print '%s is locked, but no PID file exists. breaking lock' % lockname
+            cfg.LOCK.break_lock()
```

Comment:
Would be nice to use `print` as a function here, to make the transition to py3 easier.

Reply:
done, although there are tons more places where print is still a statement.